R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+      ),
+      shiny::tags$div(tags$ul(
+        shiny::tags$li(
+          "change colours, contrast levels and fonts using browser or device settings"
+        ),
+        shiny::tags$li(
+          "zoom in up to 400% without the text spilling off the screen"
+        ),
+        shiny::tags$li(
+          "navigate most of the website using a keyboard or speech recognition software"
+        ),
+        shiny::tags$li(
+          "listen to most of the website using a screen reader
+                    (including the most recent versions of JAWS, NVDA and VoiceOver)"
+        )
+      )),
+      shiny::tags$p("We've also made the website text as simple as possible to understand."),
+      shiny::tags$p(
+        external_link(href = "https://mcmw.abilitynet.org.uk/", "AbilityNet"),
+        " has advice on making your device easier to use if you have a disability."
+      ),
+      shiny::tags$h2("How accessible this website is"),
+      if (all(is.na(non_accessible_components))) {
+        shiny::tags$p("This website is fully compliant with accessibility standards.")
+      } else {
+        shiny::tagList(
+          shiny::tags$p("We know some parts of this website are not fully accessible:"),
+          shiny::tags$div(tags$ul(
+            tagList(lapply(non_accessible_components, shiny::tags$li))
+          ))
+        )
+      },
+      shiny::tags$h2("Feedback and contact information"),
+      shiny::tags$p(
+        "If you need information on this website in a different format please see the parent",
+        "publications",
+        external_link(
+          href = "https://explore-education-statistics.service.gov.uk/find-statistics",
+          "on Explore education statistics"
+        ),
+        ", as detailed on the data sources page of this service.",
+        ". More details are available on that service for alternative formats of this data.",
+      ),
+      shiny::tags$p("We're always looking to improve the accessibility of this website.
+             If you find any problems not listed on this page or think we're not meeting
+             accessibility requirements, contact us:"),
+      shiny::tags$ul(tags$li(
+        shiny::tags$a(
+          href = "mailto:explore.statistics@education.gov.uk",
+          "explore.statistics@education.gov.uk"
+        )
+      )),
+      shiny::tags$h2("Enforcement procedure"),
+      shiny::tags$p(
+        "The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public ",
+        "Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018",
+        "(the \"accessibility regulations\")."
+      ),
+      shiny::tags$p(
+        "If you are not happy with how we respond to your complaint, ",
+        external_link(
+          href = "https://www.equalityadvisoryservice.com/",
+          "contact the Equality Advisory and Support Service (EASS)"
+        ),
+        "."
```

Comment:
To match the EES one and make it read a little easier out of context I'd make the link text just the website title, as the link doesn't directly contact them

```suggestion
        "If you are not happy with how we respond to your complaint, contact the ",
        external_link(
          href = "https://www.equalityadvisoryservice.com/",
          "Equality Advisory and Support Service (EASS)"
        ),
        "."
```